package.json

```diff
@@ -7,13 +7,14 @@
   },
   "dependencies": {
     "body-parser": "~1.13.2",
+    "bonescript": "^0.5.0-beta3",
```

Comment:
It seems ugly to have 2 copies of BoneScript sitting around and potentially have a different one used by command-line clients (like run from Cloud9 IDE) than the one used by packages (including in the bone101 demos). It seems that is the way nodejs/npm are designed. Is this the way we want to leave it?

https://github.com/npm/npm/issues/2949